ui/widget.js

```diff
@@ -246,7 +257,7 @@ $.widget.bridge = function( name, object ) {
 	};
 };
 
-$.Widget = function( /* options, element */ ) {};
+$.Widget = function( /* Options, element */ ) {};
```

Comment:
I meant I wonder if we can get JSCS to catch the scenario of an inline comment inside an argument list so that it doesn't apply the capitalization. But the other examples you've posted do make this seems very hard to enforce.